xcell/mappers/mapper_DESY1wl.py

```diff
@@ -197,7 +203,8 @@ def get_nz(self, dz=0):
     def _get_mask(self):
         cat_data = self.get_catalog()
         msk = get_map_from_points(cat_data, self.nside,
-                                  ra_name='ra', dec_name='dec')
+                                  ra_name='ra', dec_name='dec',
+                                  rot=self.rot)
```

Comment:
This is fine, but we probably want to change the lite file name to add the coordinates.